package.json

```
@@ -0,0 +1,117 @@
+{
+  "name": "dropdown",
+  "widgetName": "Dropdown",
+  "version": "1.0.0",
+  "description": "An input widget used to display and edit associations",
+  "copyright": "Mendix BV",
+  "scripts": {
+    "start": "webpack-dev-server --hot --open",
+    "emit": "tsc",
```

```suggestion
    "typeCheck": "tsc --noEmit",
```
no need to create files, it just pre-commit

---

Updated